opentelemetry/chart/templates/_helpers.tpl

```diff
@@ -12,9 +12,20 @@ plugindefinition: opentelemetry
 {{ tpl (toYaml .Values.global.commonLabels) . }}
 {{- end }}
 {{- end }}
+
 {{/* Generate prometheus specific labels */}}
-{{- define "plugin.prometheusLabels" -}}
-{{- with $.Values.openTelemetry.prometheus.additionalLabels }}
-{{- tpl (toYaml . ) $ }}
+{{- define "plugin.prometheusLabels" }}
+{{- if .Values.openTelemetry.prometheus.additionalLabels }}
+{{- tpl (toYaml .Values.openTelemetry.prometheus.additionalLabels) . }}
+{{- end }}
+{{- if .Values.prometheusRules.labels }}
+{{ tpl (toYaml .Values.prometheusRules.labels) . }}
 {{- end }}
 {{- end }}
+
+{{/* Generate prometheus rule labels for alerts */}}
+{{- define "plugin.additionalRuleLabels" -}}
+{{- if .Values.prometheusRules.additionalRuleLabels }}
+{{- tpl (toYaml .Values.prometheusRules.additionalRuleLabels) . | indent 4 }}
+{{- end }}
+{{- end }}
```

Comment:
Another indentation is not meaningful. Let's invoke the indent function in the actual alert files. 

```suggestion
{{/* Generate prometheus rule labels for alerts */}}
{{- define "plugin.additionalRuleLabels" -}}
{{- if .Values.prometheusRules.additionalRuleLabels }}
{{- tpl (toYaml .Values.prometheusRules.additionalRuleLabels) . }}
{{- end }}
{{- end }}
```